src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing nf-core office hours'
+subtitle: "A new initative from nf-core! Here's what you can expect."
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Introducing nf-core office hours'
+    start: 2024-01-12T16:14:00+01:00
+    end: 2024-01-13T16:14:00+01:00
+pubDate: 2024-01-12T16:14:00+01:00
+authors:
+    - 'christopher-hakkaart'
+label:
+    - 'community'
+---
+
+# nf-core office hours
+
+Slack serves as an outstanding platform for asynchronous collaboration.
+Nonetheless, there are moments when engaging in live discussions about your code proves beneficial.
+We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.
+
+# What you can expect
+
+Office hours will be most useful for “advanced beginners” - people who have grasped the fundamentals of writing nextflow pipelines, and are now trying to make the leap to designing, coding, and contributing nf-core components and pipelines.
+However, regardless of what stage you are at, if you think that spending some time talking about developing with nf-core sparks your interest, then the office hours might be just what you're looking for!
+
+In these sessions, some of the nf-core maintainer team and other seasoned community members will be available to engage in discussions about your code.
+They will hang out in Gather during designated hours every week, ready to provide guidance and insights into your nf-core code development.
```

Comment:
do we need to bump the gather license for this?

Reply:
I think it would be more organic in Gather but I need to check the cost and ask permission to bump it to a paid plan if everyone agrees. I was going to circulate this to the core team (and then maintainers) to ask for feedback.

Reply:
agree that it will be nice in gather, just wanted to double check the number of seats

Reply:
I think it's just 10 with current settings

Reply:
Yeah, it is 10, I've messaged inquiring about discounts for community projects.